### PR TITLE
add reporter

### PR DESCRIPTION
**Pull request by leordev**

### CODE REVIEW

File: .github/workflows/integrity-check.yml [hunk 1]
```diff
@@ -90,3 +90,35 @@ jobs:
 
       - name: Run tests for all packages
         run: pnpm test:browser
+
+  tbdocs:
```
Comment (KendallWeihe) [inline .github/workflows/integrity-check.yml:94]: could we rename this `tbdocs-warnings` as to differentiate it from the actual docs generation

File: .github/workflows/integrity-check.yml [hunk 2]
```diff
@@ -90,3 +90,35 @@ jobs:
...
+
+      - name: Build all workspace packages
+        run: pnpm build
+
+      - name: TBDocs Reporter
+        id: tbdocs-reporter-protocol
+        uses: TBD54566975/tbdocs@main
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          project_path: "./packages/protocol"
```
Comment (KendallWeihe) [inline .github/workflows/integrity-check.yml:122]: I see you have the `protocol` package, do you also want to add the `http-client` and `http-server` packages?

**Comment by KendallWeihe** [comment]
Could we update the PR title to be declarative, so like `Add tbdocs-warnings to integrity checks`?

File: .github/workflows/integrity-check.yml (hunk 2 above)
Comment (leordev) [reply .github/workflows/integrity-check.yml:122]: Yes, I need to work on supporting multiple packages/monorepo. Tracking here: https://github.com/TBD54566975/tbdocs/issues/11

**Pull request closed without merging**